README.md

```diff
@@ -9,22 +9,39 @@ Funkcjonalności
         6. Lista zakupów, po wyjściu ze strony nie musi to być nigdzie zapisane, ewntualnie sesja, ciasteczka -> Search Grocery Products
         7. Kalkulator na przeliczanie miar gramy na kilogramy i takie tam. -> Convert Amounts
 ### Przykładowe uzycia api
-1. Search Recipes  
-a. https://api.spoonacular.com/recipes/complexSearch?query=pasta&maxFat=25&number=2&apiKey=1193248e5d2944ddbd9797097eaae839&apiKey=YOUR_API_KEY  
-b.  
-c.  
+1) Search Recipes  
+                a. https://api.spoonacular.com/recipes/complexSearch&apiKey=YOUR_API_KEY <-- To jest pusty template, do którego trzeba dodać co najmniej jeden parametr (bez parametrów nie działa). 
+                
+      b. W celu dodania parametru trzeba po 'complexSearch' wpisać '?', a następnie nazwę danego parametru i jego wartość po '='.
+https://api.spoonacular.com/recipes/complexSearch?query=rice&apiKey=YOUR_API_KEY  <-- To zapytanie do API zwróci 10 obiektów, które w parametrze 'query' mają słowo 'rice'.
 
+   c. W celu dodania kilku parametrów trzeba przed każdym nowym parametrem wpisać '&'
+https://api.spoonacular.com/recipes/complexSearch?query=rice&cuisine=japanese&number=5&apiKey=YOUR_API_KEY <-- To zapytanie do API zwróci 5 obiektów, które w parametrze 'query' mają słowo 'rice', w parametrze 'cuisine' słowo 'japanese', a w parametrze 'number' liczbę '5'.
+                
+   d. Cała dokumnetacja z wszytkimi parametrami znajduje się tutaj (jest ich mnóstwo) https://spoonacular.com/food-api/docs#Search-Recipes-Complex
 
 
+2) Search Recipes by Ingredients
 
+      a. https://api.spoonacular.com/recipes/findByIngredients&apiKey=YOUR_API_KEY <-- To jest pusty template, do którego trzeba dodać co najmniej jeden parametr (bez parametrów nie działa).
 
+      b. W celu dodania parametru trzeba po 'findByIngredients' wpisać '?', a następnie nazwę danego parametru i jego wartość po '='.
+https://api.spoonacular.com/recipes/findByIngredients?ingredients=pumpkin&apiKey=YOUR_API_KEY <-- To zapytanie do API zwróci 10 obiektów, które w parametrze 'ingredients' mają słowo 'pumpkin'.
```

Comment:
Same question here?

Reply:
Well I made a mistake here. I should say "zwróci maksymalnie 10 obiektów" instead of  "zwróci 10 obiektów". The parameter "number" is set by default to 10. It is written here in the docs:

number | number | 10 | The maximum number of recipes to return (between 1 and 100). Defaults to 10.
https://spoonacular.com/food-api/docs#Search-Recipes-by-Ingredients